csharp-basics/exercises/Polymorphism/DragRace/Audi.cs

```diff
@@ -1,29 +1,22 @@
-using System;
+﻿using System;
 
 namespace DragRace
 {
-    public class Audi
+    public class Audi : Car
     {
-        private int currentSpeed = 0;
+        override public int SpeedIncrease { get;}
+        override public int SpeedDecrease { get;}
 
-        public void SpeedUp() 
+        public Audi()
         {
-            currentSpeed;
+            SpeedIncrease = 15;
```

Comment:
Kas te notiek?

Reply:
Lai nebūtu katrā klasē jāliek SpeedUp un SlowDown metodes tikai, lai pamainītu to pa cik mašīnai jāpaātrinās vai jāsamazina ātrums, tad es izveidoju vienkārši Properties SpeedIncrease un SpeedDecrease, tie tad tiek izmantoti bāzes klasēs lai noteiktu pa cik paātrināties vai samazināt ātrumu

Reply:
hmm, ok